frontend/src/metabase/app.js

```diff
@@ -91,7 +89,7 @@ function _init(reducers, getRoutes, callback) {
 
   store.dispatch(refreshSiteSettings());
 
-  PLUGIN_APP_INIT_FUCTIONS.forEach(init => init({ root }));
+  PLUGIN_APP_INIT_FUCTIONS.forEach(init => init());
```

Comment:
There's only one function (`updateColors`) that goes to `PLUGIN_APP_INIT_FUCTIONS` and it does not use root ref.